docs/node/meta-package.md

```diff
@@ -0,0 +1,5 @@
+# temporalio
```

Comment:
This page/info is a little island right now - Seems like it should be in the introduction or something? I am not a fan of having this as a stand alone page.

Reply:
Hmm.. I didn't really not a lot to say, but this needs to be documented and it fits the structure.